src/main/java/ca/gc/aafc/collection/api/dto/MaterialSampleDto.java

```diff
@@ -128,10 +97,40 @@ public class MaterialSampleDto {
   private HostOrganism hostOrganism;
   
   @IgnoreDinaMapping
-  private List<AssociationDto> associations = new ArrayList<>();
+  private List<AssociationDto> associations = List.of();
 
   private Boolean allowDuplicateName = false;
 
+  private List<ExtensionValue> restrictionFieldsExtension;
+
+  private Boolean isRestricted = false;
+
+  private String restrictionRemarks;
+
+  // -- Relationships --
+
+  @JsonApiRelation
+  private MaterialSampleDto parentMaterialSample;
+
+  @JsonApiRelation
+  private CollectingEventDto collectingEvent;
+
+  @JsonApiRelation
+  private CollectionDto collection;
+
+  @JsonApiRelation
+  private PreparationTypeDto preparationType;
+
+  @JsonApiRelation
+  private PreparationMethodDto preparationMethod;
+
+  @JsonApiRelation
+  private StorageUnitDto storageUnit;
+
+  @JsonApiRelation
+  @JsonInclude(JsonInclude.Include.NON_EMPTY)
+  private List<OrganismDto> organism = List.of();
```

Comment:
I like that this is organized now, much easier to find what you are looking for.